src/lib/Stats.ts

```diff
@@ -0,0 +1,25 @@
+import * as StatsD from 'hot-shots';
+
+const { MICROS_ENV, MICROS_ENVTYPE, NODE_ENV } = process.env;
+
+let client;
+
+if (!MICROS_ENV || !MICROS_ENVTYPE) {
+  client = {
+    increment: (stat: string, tags?: StatsD.Tags) => {},
+  };
+} else {
+  const HOST = NODE_ENV === 'production' ? 'platform-statsd' : 'localhost';
+
+  client = new StatsD.StatsD({
+    host: HOST,
+    port: 8125,
+    prefix: 'atlassian_frontend_landkid.',
```

Comment:
Just double checking, is it normal to have the trailing `.` here?

Reply:
I forgot to change that but will fix it up in the next commit with the update duration metric name.